CalibCalorimetry/HcalAlgos/interface/HcalHardcodeParameters.h

```diff
@@ -11,7 +11,7 @@ class HcalHardcodeParameters {
 		//default constructor
 		HcalHardcodeParameters() {}
 		//construct from values
-		HcalHardcodeParameters(double pedestal, double pedestalWidth, std::vector<double> gain, std::vector<double> gainWidth, 
+		HcalHardcodeParameters(double pedestal, double pedestalWidth, std::vector<double> gain, std::vector<double> gainWidth, int zsThreshold,

```

Comment:
You are not changing it here, but you should pass the `std::vector`s by const reference, although the compiler might optimize this away, as you anyway copy it into a data member.